rules/solve_overnight.smk

```diff
@@ -3,6 +3,32 @@
 # SPDX-License-Identifier: MIT
 
 
+def ptes_operation_profiles(w):
+    """
+    Return a dict of only the PTES profiles that are enabled in config,
+    keyed by the same names you’d have used in `input:`
+    """
+    profiles = {}
+    if config_provider(
+        "sector", "district_heating", "ptes", "discharge_boosting_required"
+    )(w):
+        profiles["boost_per_discharge_profile"] = resources(
+            "boost_per_discharge_profile_base_s_{clusters}_{planning_horizons}.nc"
+        )
+        profiles["cop_profiles"] = resources(
+            "cop_profiles_base_s_{clusters}_{planning_horizons}.nc"
+        )
+
+    if config_provider(
+        "sector", "district_heating", "ptes", "charge_boosting_required"
+    )(w):
+        profiles["boost_per_charge_profile"] = resources(
+            "boost_per_charge_profile_base_s_{clusters}_{planning_horizons}.nc"
+        )
+
+    return profiles
```

Comment:
Can move to `common.smk` to prevent code duplication of this function in overnight and myopic scenarios.